mnist/README.md

```diff
@@ -0,0 +1,172 @@
+# Mnist
+
+*This example is a Substra implementation of on [the MNIST example from Keras](https://keras.io/examples/vision/mnist_convnet/), inspired from [Substra's Titanic Example](https://github.com/SubstraFoundation/substra/blob/master/examples/titanic/)*
+
+In this example, we'll see how to setup an objective and how to train algorithms on the MNIST dataset.
+## Prerequisites
+
+In order to run this example, you'll need to:
+
+* use Python 3
+* have [Docker](https://www.docker.com/) installed
+* [install the `substra` cli](https://github.com/SubstraFoundation/substra#install)
+* [install the `substratools` library](https://github.com/substrafoundation/substra-tools)
+* [pull the `substra-tools` docker images](https://github.com/substrafoundation/substra-tools#pull-from-private-docker-registry)
+* create a substra profile to define the substra network to target, for instance:
+    ```sh
+    substra config --profile node-1 http://substra-backend.node-1.com
+    substra login --profile node-1 --username node-1 --password 'p@$swr0d44'
```

Comment:
So a pre-requisite is to have access to an existing Substra installation. I think this should be made explicit :)

Reply:
So I could add something like: 
* having access to a Substra installation ([configure your host to a public node ip](https://doc.substra.ai/getting_started/installation/local_install_skaffold.html#network) or [install Substra on your machine](https://doc.substra.ai/getting_started/installation/local_install_skaffold.html))

The prerequisites part is a copypaste from Titanic, so it should be made there too ? I could do a PR there too.